doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -65,18 +65,18 @@ endif::[]
 endif::offline_onprem[]
 . On the computer that has an outgoing connection to the public Internet, complete the following steps:
 .. Download the latest version of the http://release-engineering.github.io/offliner/[Offliner tool].
-.. Download the `{PRODUCT_FILE}-offliner.txt` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
-.. Enter the following command to use the Offliner tool to download the required artifacts:
+.. Download the `{PRODUCT_FILE}-offliner.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
```

Comment:
I know this isn't something you changed, but this isn't the way we should refer to links:
Download the latest version of the Offliner tool.

You don't need to change it for the release, but just so you know, I would do something like:
Download the latest version of the Offliner tool from the http://release-engineering.github.io/offliner/[Offliner @RedHat] website.

Reply:
This part was actually removed from this section, but I will note it for the future, thanks.